cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/export/StateVariablesExport.java

```diff
@@ -223,14 +223,20 @@ private static void writePowerFlows(Network network, String cimNamespace, XMLStr
         network.getBranchStream().forEach(b -> {
             writePowerFlowTerminalFromAlias(b, Conversion.CGMES_PREFIX_ALIAS_PROPERTIES + CgmesNames.TERMINAL1, b.getTerminal1().getP(), b.getTerminal1().getQ(), cimNamespace, writer, context);
             writePowerFlowTerminalFromAlias(b, Conversion.CGMES_PREFIX_ALIAS_PROPERTIES + CgmesNames.TERMINAL2, b.getTerminal2().getP(), b.getTerminal2().getQ(), cimNamespace, writer, context);
-            if (b instanceof TieLine && context.exportBoundaryPowerFlows()) {
+            if (b instanceof TieLine) {
                 TieLine tl = (TieLine) b;
-                writePowerFlowTerminalFromProperty(tl, Conversion.CGMES_PREFIX_ALIAS_PROPERTIES + tl.getHalf1().getId() + ".Terminal_Network",
-                        tl.getTerminal1().getP(), tl.getTerminal1().getQ(), cimNamespace, writer, context);
-                writePowerFlowTerminalFromProperty(tl, Conversion.CGMES_PREFIX_ALIAS_PROPERTIES + tl.getHalf2().getId() + ".Terminal_Network",
-                        tl.getTerminal2().getP(), tl.getTerminal2().getQ(), cimNamespace, writer, context);
-                writePowerFlowTerminalFromAlias(tl, Conversion.CGMES_PREFIX_ALIAS_PROPERTIES + "HALF1." + CgmesNames.TERMINAL + "_Boundary", tl.getHalf1().getBoundary().getP(), tl.getHalf1().getBoundary().getQ(), cimNamespace, writer, context);
-                writePowerFlowTerminalFromAlias(tl, Conversion.CGMES_PREFIX_ALIAS_PROPERTIES + "HALF2." + CgmesNames.TERMINAL + "_Boundary", tl.getHalf2().getBoundary().getP(), tl.getHalf2().getBoundary().getQ(), cimNamespace, writer, context);
+                if (!b.hasProperty(Conversion.CGMES_PREFIX_ALIAS_PROPERTIES + CgmesNames.TERMINAL1)) { // prevent duplicates
```

Comment:
the lines above have written the flows if the **alias** `TERMINAL1/2` exist. And here we are checking for a **property** named `TERMINAL1/2`. If we want to avoid duplicates, shouldn't we check for the alias instead?

Reply:
No because for tie lines, both aliases and properties are added (with the same values). Properties are added to be consistent with merging (merging of aliases is not handled in destructive merge or in merging view).

Reply:
done